lm_eval/models/huggingface.py

```diff
@@ -239,6 +239,13 @@ def __init__(
             if self.config.model_type == "qwen":
                 # Qwen's trust_remote_code tokenizer does not allow for adding special tokens
                 self.tokenizer.pad_token = "<|endoftext|>"
+            elif (self.config.model_type == "rwkv4" or self.config.model_type == "rwkv5") and self.tokenizer.__class__.__name__ == "RWKVWorldTokenizer":
```

Comment:
Might it be possible that non-RWKV models adopt this tokenizer? if so, we can remove the model_type part of this condition and just check class name

Reply:
@haileyschoelkopf That's a good idea.